### ISSUE TITLE
align_double_arrow looks weird

### ISSUE THREAD

**Issue by OskarStark** [FriendsOfPHP/PHP-CS-Fixer#1450]
Since the last update ``align_double_arrow``looks weird

```bash
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$ make cs
./vendor/bin/php-cs-fixer fix --verbose
Loaded config from "/Volumes/development/workspaces/SonataBlockBundle/.php_cs"
.................FF...........................................................................................
Legend: ?-unknown, I-invalid file syntax, file ignored, .-no changes, F-fixed, E-error
   1) Block/Service/MenuBlockService.php (align_double_arrow)
   2) Block/Service/RssBlockService.php (align_double_arrow)
Fixed all files in 23.432 seconds, 9.500 MB memory used
make: *** [cs] Error 1
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$ git diff
diff --git a/Block/Service/MenuBlockService.php b/Block/Service/MenuBlockService.php
index 6cd0965..2c88923 100644
--- a/Block/Service/MenuBlockService.php
+++ b/Block/Service/MenuBlockService.php
@@ -133,16 +133,16 @@ class MenuBlockService extends BaseBlockService
     protected function getFormSettingsKeys()
     {
         return array(
-            array('title', 'text', array('required' => false)),
-            array('cache_policy', 'choice', array('choices' => array('public', 'private'))),
-            array('menu_name', 'choice', array('choices' => $this->menus, 'required' => false)),
+            array('title', 'text', array('required'           => false)),
+            array('cache_policy', 'choice', array('choices'   => array('public', 'private'))),
+            array('menu_name', 'choice', array('choices'      => $this->menus, 'required' => false)),
             array('safe_labels', 'checkbox', array('required' => false)),
-            array('current_class', 'text', array('required' => false)),
-            array('first_class', 'text', array('required' => false)),
-            array('last_class', 'text', array('required' => false)),
-            array('menu_class', 'text', array('required' => false)),
-            array('children_class', 'text', array('required' => false)),
-            array('menu_template', 'text', array('required' => false)),
+            array('current_class', 'text', array('required'   => false)),
+            array('first_class', 'text', array('required'     => false)),
+            array('last_class', 'text', array('required'      => false)),
+            array('menu_class', 'text', array('required'      => false)),
+            array('children_class', 'text', array('required'  => false)),
+            array('menu_template', 'text', array('required'   => false)),
         );
     }

diff --git a/Block/Service/RssBlockService.php b/Block/Service/RssBlockService.php
index 80e906e..be76f09 100644
--- a/Block/Service/RssBlockService.php
+++ b/Block/Service/RssBlockService.php
@@ -51,7 +51,7 @@ class RssBlockService extends BaseBlockService
     {
         $formMapper->add('settings', 'sonata_type_immutable_array', array(
             'keys' => array(
-                array('url', 'url', array('required' => false)),
+                array('url', 'url', array('required'    => false)),
                 array('title', 'text', array('required' => false)),
             ),
         ));
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$ vendor/bin/php-cs-fixer --version
PHP CS Fixer version 1.10.1 by Fabien Potencier
oskar.stark:/Volumes/development/workspaces/SonataBlockBundle (master)$
```

**Comment by OskarStark** [comment]
``.php_cs`` file:
```php
<?php

$finder = Symfony\CS\Finder\DefaultFinder::create()
    ->in(array(__DIR__))
;

return Symfony\CS\Config\Config::create()
    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
    ->fixers(array(
        '-unalign_double_arrow',
        '-unalign_equals',
        'align_double_arrow',
        'newline_after_open_tag',
        'ordered_use',
        'long_array_syntax',
    ))
    ->setUsingCache(true)
    ->finder($finder)
;
```

**Comment by GrahamCampbell** [comment]
Looks like a bug with nested arrays.

**Comment by OskarStark** [comment]
think so, too

**Comment by OskarStark** [comment]
ping @Soullivaneuh

**Comment by Soullivaneuh** [comment]
Regarding the changelog, should be related to #1419 or #1422.

**Comment by SpacePossum** [comment]
My guess it is https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1422
If I understand correctly, this:
```php
         $formMapper->add('settings', 'sonata_type_immutable_array', array(
             'keys' => array(
                 array('url', 'url', array('required' => false)),
                 array('title', 'text', array('required' => false)),
             ),
         ));
```

is no changed to:
```php
         $formMapper->add('settings', 'sonata_type_immutable_array', array(
             'keys' => array(
                 array('url', 'url', array('required'    => false)),
                 array('title', 'text', array('required' => false)),
             ),
         ));
```

but you don't want the `align_double_arrow` fixer to do this alignment?

**Comment by GrahamCampbell** [comment]
I'd say that sort of alignment on a singleline array is a bug.

**Comment by Soullivaneuh** [comment]
> I'd say that sort of alignment on a singleline array is a bug.

:+1:

**Comment by OskarStark** [comment]
@SpacePossum 

> is no changed to:
is NOW changed to

yes, this happens

**Comment by Soullivaneuh** [comment]
This is a very annoying bug affecting a lot of projects.

Maybe can we revert the concerned PR on a new patch version to fix it later? cc @SpacePossum @keradus

**Comment by SpacePossum** [comment]
I cannot fix this within the upcoming days, so if no one else can pick it up feel free to revert the change,

**Comment by OskarStark** [comment]
is this now fixed @Soullivaneuh

**Comment by Soullivaneuh** [comment]
> is this now fixed @Soullivaneuh

With my revert yes: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1455

But nothing officially merged/fixed for now.

**Comment by OskarStark** [comment]
:+1:  ok

**Comment by OskarStark** [comment]
Can anybody please merge the revert PR of @Soullivaneuh 

This bug is really annoying

**Comment by keradus** [comment]
Ref #1460

**Comment by keradus** [comment]
Closing due to merge of #1460

**Comment by Soullivaneuh** [comment]
Not totally resolved, si #1463.
